### PR TITLE
Add support for MS Api v2

### PR DESCRIPTION
**Pull request by rzukow**

### CODE REVIEW

File: scribejava-apis/src/main/java/com/github/scribejava/apis/service/MicrosoftAzureActiveDirectoryService.java [hunk 1]
```diff
@@ -8,18 +8,18 @@
...
     public MicrosoftAzureActiveDirectoryService(DefaultApi20 api, String apiKey, String apiSecret, String callback,
             String scope, String state, String responseType, String userAgent, HttpClientConfig httpClientConfig,
-            HttpClient httpClient) {
+            HttpClient httpClient, String acceptedFormat) {
         super(api, apiKey, apiSecret, callback, scope, state, responseType, userAgent, httpClientConfig, httpClient);
+        this.acceptedFormat = acceptedFormat;
     }
 
     @Override
     public void signRequest(String accessToken, OAuthRequest request) {
```
Comment (rzukow) [inline scribejava-apis/src/main/java/com/github/scribejava/apis/service/MicrosoftAzureActiveDirectoryService.java]: This method isn't good example of clean code. Why we set a header in a method called "signRequest". I didn't change that though...

Comment (kullfar) [reply scribejava-apis/src/main/java/com/github/scribejava/apis/service/MicrosoftAzureActiveDirectoryService.java]: Hm. You're right, isn't very clean

**Comment by kullfar** [comment]
Hi,
Thanks for the Pull Request.
I need test app credentials and test user credentials to run the example.

**Comment by kullfar** [comment]
As for code, look good, thx

**Comment by rzukow** [comment]
Do you need help with this? I tested both examples with credentials I have but these belong to my employer.

**Comment by kullfar** [comment]
Yep, I need these test credentials from you.
You can register new, fake, just for test. (c) https://github.com/scribejava/scribejava/wiki/FAQ#when-will-you-support-insert_api_here-

Thanks in advance

**Comment by kullfar** [comment]
still waiting for credentials

**Comment by kullfar** [comment]
outdated

**Comment by kullfar** [comment]
have changed a bit
have adopted to the new master
have tested (thanks to @dgrudenic)
have merged to the master (thanks to @rzukow)

**Pull request closed without merging**